### PR TITLE
Feature: add default exchange support

### PR DESCRIPTION
**Pull request by stepanho**
See [docs](https://www.rabbitmq.com/tutorials/amqp-concepts.html#exchange-default) for explanation.

### CODE REVIEW

**Comment by mkls** [comment]
it looks good, but I would also be okay with a single PR for all these changes. (it would reduce the possibility of two of them conflicting).
Let me know when you think it is ready, and I will merge it. (Cannot merge it currently as it is in draft status)

**Comment by stepanho** [comment]
@mkls thank you!
I've checked possibility of conflicts, looks like possible to merge one-by-one.
I'll make some final lookups and come back with "ready" status.

File: src/main.spec.js [hunk 1]
```diff
@@ -97,6 +97,23 @@ test('purgeQueue deletes messages from queue', async () => {
   expect(message).toEqual(false);
 });
 
+test('default exchange', async () => {
+  const connection = await amqp.connect('some-random-uri');
+  const channel = await connection.createChannel();
+  await channel.assertQueue('retry-queue-10s');
+  await channel.assertQueue('retry-queue-20s');
+  await channel.publish('', 'retry-queue-10s', 'content-1');
```
Comment (sempasha) [inline src/main.spec.js]: Should also we test `sendToQueue` method here?

Also `sendToQueue` might use `publish` to defult exchange at backend as well.
Something like

```js
{
  async publish(/*publish params*/) {
    /* publics implementation */
  },
  async sendToQueue(queueName, content, { headers } = {}) {
    this.publish(DEFAULT_EXCHANGE_NAME, routingKey, content, { headers });
  },
}
```

File: src/main.spec.js [hunk 2]
```diff
@@ -97,6 +97,23 @@ test('purgeQueue deletes messages from queue', async () => {
   expect(message).toEqual(false);
 });
 
+test('default exchange', async () => {
+  const connection = await amqp.connect('some-random-uri');
+  const channel = await connection.createChannel();
+  await channel.assertQueue('retry-queue-10s');
```
Comment (sempasha) [inline src/main.spec.js]: Whould be betrer to use test related queue names. Prefix `retry-` already used by tests below. So we may use something like `default-queue` instead.

**Review by sempasha — APPROVED** [review]
@stepanho well done! Wait for library update.

File: src/main.spec.js (hunk 1 above)
Comment (stepanho) [reply src/main.spec.js]: great suggestion, changed

File: src/main.spec.js (hunk 2 above)
Comment (stepanho) [reply src/main.spec.js]: used declared functions

**Comment by stepanho** [comment]
Hi @mkls!
> it looks good, but I would also be okay with a single PR for all these changes. (it would reduce the possibility of two of them conflicting).

Just merge one-by-one, if conflict appears - I will fix it.
> Let me know when you think it is ready, and I will merge it. (Cannot merge it currently as it is in draft status)

All PRs now in _ready_ state.

**Pull request merged**